roborazzi-gradle-plugin/src/main/java/io/github/takahirom/roborazzi/RoborazziPlugin.kt

```diff
@@ -23,6 +23,7 @@ class RoborazziPlugin : Plugin<Project> {
     val verifyVariants = project.tasks.register("verifyRoborazzi")
     val compareVariants = project.tasks.register("compareRoborazzi")
     val recordVariants = project.tasks.register("recordRoborazzi")
+    val verifyAndRecordVariants = project.tasks.register("verifyAndRecordRoborazzi")
```

Comment:
Looks fine but we don't use Gradle so I'd have someone else review if you think it needs a set of eyes.

Reply:
Thanks. I don't think it's a big change to the Gradle tasks. So we can fix it if there's anything.